### PR TITLE
[tizen_app_control] Support getMatchedAppIds

### PR DESCRIPTION
**Pull request by swift-kim**
Related to https://github.com/flutter-tizen/engine/pull/185.

Reference: TizenFX's [`AppControl.GetMatchedApplicationIds()`](https://docs.tizen.org/application/dotnet/api/TizenFX/API8/api/Tizen.Applications.AppControl.html#Tizen_Applications_AppControl_GetMatchedApplicationIds_Tizen_Applications_AppControl_)

### CODE REVIEW

File: packages/tizen_app_control/example/integration_test/tizen_app_control_test.dart [hunk 1]
```diff
@@ -26,6 +26,16 @@ void main() {
...
 
+  testWidgets('Can find matching applications', (WidgetTester _) async {
+    final AppControl request = AppControl(
+      operation: 'http://tizen.org/appcontrol/operation/view',
+      uri: 'file:///opt/usr/globalapps/$kAppId/shared/res/ic_launcher.png',
+      mime: 'image/*',
+    );
+    final List<String> matches = await request.getMatchedAppIds();
+    expect(matches, isNotEmpty);
+  }, timeout: kTimeout);
```
Comment (swift-kim) [inline packages/tizen_app_control/example/integration_test/tizen_app_control_test.dart:37]: This test should not be run on TV emulators. I couldn't find any operation that is commonly available on both wearable and TV emulators.

Comment (WonyoungChoi) [reply packages/tizen_app_control/example/integration_test/tizen_app_control_test.dart:37]: how about tagging this test to mark "wearable only"?
https://pub.dev/packages/test#tagging-tests

**Review by WonyoungChoi — APPROVED** [review]
LGTM

File: packages/tizen_app_control/example/integration_test/tizen_app_control_test.dart (hunk 1 above)
Comment (swift-kim) [reply packages/tizen_app_control/example/integration_test/tizen_app_control_test.dart:37]: Could you elaborate how to configure? How can the test runner detect the current device profile at runtime?

**Pull request merged**